src/Fable.ReactGoogleMaps/js/mapComponent.js

```diff
@@ -0,0 +1,91 @@
+import { withScriptjs, withGoogleMap, GoogleMap, TrafficLayer, Marker } from "react-google-maps";
+const { SearchBox } = require("react-google-maps/lib/components/places/SearchBox");
+import React, {Component, Children} from 'react';
+
+const TrafficMapComponent = withScriptjs(withGoogleMap((props) =>
+  <GoogleMap
```

Comment:
Is there a way to get rid of JSX? Can we rewrite this?

Reply:
Yes, it'll be better so users are not forced to include the Babel react-preset. It's really easy to replace JSX, you can use the [Babeljs repl](https://babeljs.io/repl#?babili=false&browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=DwMQ9mAEBmELwCIBGBDATgyB6AfEA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=true&fileSize=true&timeTravel=false&sourceType=module&lineWrap=false&presets=react%2Cstage-2%2Cstage-3%2Ces2015-loose%2Cenv&prettier=true&targets=Node-8.9&version=6.26.0&envVersion=1.6.2) to see the corresponding JS, but basically `<Foo foo="foo" bar={5}><Child1 /><Child2 /></Foo>` becomes:

```js
React.createElement(
  Foo,
  { foo: "foo", bar: 5 },
  React.createElement(Child1, null),
  React.createElement(Child2, null)
);
```

Reply:
Another option is to just pass this file through babel-cli to remove the JSX before packaging. I need to check but something like the following should work ([docs](https://babeljs.io/docs/en/babel-cli)). But remember to adjust the import path in the F# file:

```
npm install --save-dev @babel/core @babel/cli @babel/preset-react
npx babel script.js --out-file script-compiled.js --presets=react
```

Reply:
that would be awesome solution!

Reply:
ok got rid of the JSX. thanks!